medusa/name_parser/parser.py

```diff
@@ -461,6 +466,16 @@ def get(self, name):
             logger.debug("Using cached parse result for '{name}'", name=name)
             return self.cache[name]
 
+    def remove(self, indexer, indexer_id):
+        """Remove cache item given indexer and indexer_id."""
+        if not indexer or not indexer_id:
+            return
+        to_remove = (cached_name for cached_name, cached_parsed_result in self.cache.items() if
```

Comment:
need to loop as there is no other way. I can't `pop` a item without knowing the name of it
Also I'm using generator

I don't see why is bad to loop. 
How often a user change scene numbering? once a week or few days?
This is a task that will only run when user does some action. Not a scheduled action